arm-software/linux/CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [20.1.0]
+
+This is the first release of the Arm Toolchain for Linux (ATfL), a successor of
+the Arm Compiler for Linux.
+
+Although it is entirely based on the LLVM project, there were some changes
```

Comment:
Do we want to mention at least which main release we're basing this on (maybe it's covered elsewhere, so not required?)